crates/chainio/clients/avsregistry/src/fake_reader.rs

```diff
@@ -45,7 +43,7 @@ impl AvsRegistryReader for FakeAvsRegistryReader {
         Ok(vec![vec![OperatorStateRetriever::Operator {
             operator: self.operator_address,
             operatorId: self.operator_id,
-            stake: OPERATOR_STAKE,
+            stake: U96::from(123),
```

Comment:
why did you remove the constant?

Reply:
because  it required use of lazy constant and was used only once. shoudl i use lazy?